deftree/build_deftree.go

```diff
@@ -101,24 +101,21 @@ func NewFromString(def string) (Deftree, error) {
 	}
 	defer os.RemoveAll(protoDir)
 
-	err = ioutil.WriteFile(filepath.Join(protoDir, defFileName), []byte(def), 0666)
-	if err != nil {
-		return nil, errors.Wrap(err, "could not write proto definition to file")
-	}
+	defPath := filepath.Join(protoDir, defFileName)
 
-	err = protostage.Stage(protoDir)
+	err = ioutil.WriteFile(defPath, []byte(def), 0666)
```

Comment:
On the 'all literals should be constants' debate I stray towards defining constants when you actually start using the value in multiple places. We don't appear to be here, which makes this OK in my book, but we should still leave a comment with some indication on why these permissions where chosen. Especially since they do not match the permission we use in protostuff/mkdir()